stack/README.md

```diff
@@ -1,3 +1,65 @@
-# stack
+# Couchbase Stack Deployment
 
-These are ARM templates for Azure Stack.
+## Usages:
+
+To perform a deployment execute:
+
+    ./deploy.sh -l <LOCATION> -p ./mainTemplateParameters.json -g <RESOURCE_GROUP_NAME> -n <DEPLOYMENT_NAME>
+
+To remove a deployment execute:
+
+    ./backout.sh -g <RESOURCE_GROUP_NAME>
+
+optional -s parameter will prevent the removal of the resource group.  This operation can take a while to execute.
+
+## Template Parameters
+
+    /* The number of nodes to deploy */
+    "serverNodeCount": {
+      "type": "int",
+      "defaultValue": 3
+    },
+    /* The size of the HDD */
```

Comment:
(minor) might want to include in comment this is in GB just in case someone gets a crazy idea to start a vm with a 4096 hd

Reply:
done